iml-manager-cli/src/main.rs

```diff
@@ -3,12 +3,15 @@
 // license that can be found in the LICENSE file.
 
 use futures::Future;
+
 use iml_manager_cli::{
     api_utils,
     manager_cli_error::{DurationParseError, ImlManagerCliError},
 };
+
 use iml_wire_types::{ApiList, Command, Host};
 use prettytable::{Row, Table};
+use reqwest as _;
```

Comment:
Dead code?

Reply:
I think it's being used as my compiler isn't issuing a warning.

Reply:
I don't see how it could be... it's not a trait.

Reply:
I removed it and I don't see any warnings or errors.